model/fv_arrays.F90

```diff
@@ -916,6 +916,12 @@ module fv_arrays_mod
   real(kind=R_GRID) :: deglat=15.   !< Latitude (in degrees) used to compute the uniform f-plane
                                     !< Coriolis parameter for doubly-periodic simulations
                                     !< (grid_type = 4). The default value is 15.
+  real(kind=R_GRID) :: deglon=0.    !< Longitude (in degrees) used for radiation
+                                    !< calculations for doubly-periodic simulations.
+                                    !< (grid_type = 4). Default value is 0.
+  real(kind=R_GRID) :: deg_domain=10.  !< Domain size (north-south & east-east) in
```

Comment:
Should this be set to 11.25 to maintain original 'arbitrary' value of pi/16? (Otherwise will now be pi/18)